nodejs/eks/nodegroup.ts

```diff
@@ -494,10 +546,12 @@ ${customUserData}
         securityGroups: [nodeSecurityGroupId, ...extraNodeSecurityGroupIds],
         spotPrice: args.spotPrice,
         rootBlockDevice: {
-            encrypted: args.encryptRootBlockDevice || args.encryptRootBockDevice,
-            volumeSize: args.nodeRootVolumeSize || 20, // GiB
-            volumeType: "gp2", // default is "standard"
-            deleteOnTermination: true,
+            encrypted: (args.encryptRootBlockDevice ?? args.nodeRootVolumeEncrypted) ?? false,
+            volumeSize: args.nodeRootVolumeSize ?? 20, // GiB
```

Comment:
This is a semantic change if the value is `0`, but presumably that wouldn't have been a valid input anyway, so I think this is ok.